cmake/FindIntel_SYCL.cmake

```diff
@@ -11,7 +11,13 @@ if(NOT DEFINED INTEL_SYCL_TRIPLE)
 endif()
 message("Intel SYCL target triple: ${INTEL_SYCL_TRIPLE}")
 
-set(INTEL_SYCL_FLAGS "-fsycl;-fsycl-targets=${INTEL_SYCL_TRIPLE};-sycl-std=2020;-ffp-model=precise;${INTEL_SYCL_FLAGS}")
+if(WIN32)
+    set(INTEL_FP_FLAG "/fp:precise") # clang-cl fp flag for windows
+else()
+    set(INTEL_FP_FLAG "-ffp-model=precise")
+endif()
+
+set(INTEL_SYCL_FLAGS "-fsycl;-fsycl-targets=${INTEL_SYCL_TRIPLE};-sycl-std=2020;${INTEL_FP_FLAG};${INTEL_SYCL_FLAGS}")
```

Comment:
INTEL_SYCL_FLAGS are not used for oclmath where fp-model precise is needed, Maybe it's better to add this flag to CMAKE_CXX_FLAGS?

Reply:
Thanks! I'll update it tomorrow.
+@bader